### ISSUE TITLE
Issues with borders and resetting styles in Firefox Developer Edition

### ISSUE THREAD

**Issue by jillianadriana** [soflyy/oxygen-bugs-and-features#2939]
I haven't actually tried in other browsers, but in Firefox Developer Edition, trying to set borders and also hitting the little reset icon to clear styles from a property brings up a toolbar that says "This page is slowing down Firefox Developer Edition. To speed up your browser, stop this page", with buttons to either stop it or debug it.

When setting borders, this typically happens when I am trying to type in my color var, and I have to wait for this process after every single letter I type (or have to backspace and re-type).

**Comment by MattHag** [comment]
Yes, the border controls are totally useless in its current state with Firefox Dev. I can't even enter a single character in the color input - maybe after an hour it's there?

**Comment by derSebastian** [comment]
Can confirm that this is happening in the std version of Firefox as well. In Chrome/Brave this is also happening but without alert. It just takes a long time. In Safari this is also a problem.

**Comment by KittenCodes** [comment]
This will be fixed in v4.0.3.
